examples/bq_file_load_benchmark/benchmark_tools/benchmark_result_util.py

```diff
@@ -1,284 +0,0 @@
-# Copyright 2018 Google Inc.
```

Comment:
Is it intended that this file  was deleted as part of this python3 migration?

Reply:
nope that should have been done in an earlier PR. I just noticed it here and deleted it.

Reply:
This is going to remove this file from master if we merge.
Is that what you mean to do?

Reply:
yes